client/ayon_core/style/__init__.py

```diff
@@ -117,7 +117,7 @@ def get_objected_colors(*keys):
 
 
 def _load_stylesheet():
-    """Load strylesheet and trigger all related callbacks.
+    """Load stylesheet and trigger all related callbacks.
```

Comment:
Not sure if it forth fork just for a description typo

Reply:
typo restored